Classes/Errors/TMAPIError.h

```diff
@@ -46,6 +46,11 @@
  */
 @property (nonatomic, readonly) BOOL needsAge;
 
+/**
+ A specific property associated with some errors for the GUCE Consent flow. This specifies the age the user must be based on their gdpr region.
+ */
+@property (nonatomic, readonly) NSInteger gdprMinimumAge;
```

Comment:
`gdprMinimumAge` - I was thinking more like `minimumRequiredAge` or something `GDPR` is super vague